Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/CheckVersionIntegrityCommand.cs

```diff
@@ -8,8 +8,15 @@
 
 namespace PatchKit.Unity.Patcher.AppUpdater.Commands
 {
+    public enum IntegrityLevel
```

Comment:
I'd prefer to use flags here.

Reply:
Any substantiation?

Reply:
Currently we have two options. If we add a third we would need to create multiple combinations "first_and_second", "second_and_third", "only_first" etc.

We can solve this by using flags here.

Reply:
Tried to avoid flags because flags imply branching and not doing one thing (SOLID). That is why tried doing it with enumerations as levels according to current specification. We can do it as frags if you really want it. I am not a big fan of flags because flags imply sort of special cases in in code that inmost of cases can be better designed with ex. enums or strategy pattern.